lib/core/src/lib/comments/comments.component.ts

```diff
@@ -46,141 +51,127 @@ export class CommentsComponent implements OnChanges {
     @Output()
     error: EventEmitter<any> = new EventEmitter<any>();
 
-    comments: CommentModel [] = [];
-
-    private commentObserver: Observer<CommentModel>;
-    comment$: Observable<CommentModel>;
+    comments: CommentModel[] = [];
 
     message: string;
 
     beingAdded: boolean = false;
 
-    constructor(private commentProcessService: CommentProcessService,
-                private commentContentService: CommentContentService) {
+    private commentObserver: Observer<CommentModel>;
+    comment$: Observable<CommentModel>;
+
+    constructor(@Inject(ADF_COMMENTS_SERVICE) private commentsService: CommentsService) {
         this.comment$ = new Observable<CommentModel>((observer) => this.commentObserver = observer)
-            .pipe(share());
+            .pipe(
+                share()
+            );
+
         this.comment$.subscribe((comment: CommentModel) => {
             this.comments.push(comment);
         });
     }
 
-    ngOnChanges(changes: SimpleChanges) {
-        this.taskId = null;
-        this.nodeId = null;
+    ngOnChanges(changes: SimpleChanges): void {
+        this.id = null;
 
-        this.taskId = changes['taskId'] ? changes['taskId'].currentValue : null;
-        this.nodeId = changes['nodeId'] ? changes['nodeId'].currentValue : null;
+        this.id = changes['id'] ? changes['id'].currentValue : null;
 
-        if (this.taskId || this.nodeId) {
+        if (this.id) {
             this.getComments();
         } else {
             this.resetComments();
         }
     }
 
-    private getComments(): void {
+    getComments() {
```

Comment:
can this be `loadComments()` instead? the "get" implies some return value will be provided as a function call...